sbt-play-ebean/src/main/scala/play/ebean/sbt/PlayEbean.scala

```diff
@@ -25,7 +40,24 @@ object PlayEbean extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  override def projectSettings = inConfig(Compile)(scopedSettings) ++ unscopedSettings
+  override def projectSettings = {
+    val querySettings = Seq(
+      playEbeanQueryGenerate := false,
+      playEbeanQueryEnhance := false,
+      playEbeanQueryDestDirectory := "app",
+      playEbeanQueryResourceDirectory := "conf",
+      playEbeanQueryModelsPackage := "models",
```

Comment:
It should be configurable.
If the user specifies playEbeanQueryModelsPackage := "notmodels" in their build.sbt it will override the default models definition unless I'm not understanding your comment correctly.